binding_generator.py

```diff
@@ -346,7 +346,7 @@ def generate_class_implementation(icalls, used_classes, c):
                 if is_enum(method["return_type"]):
                     return_statement += "return (" + remove_enum_prefix(method["return_type"]) + ") "
                 elif is_reference_type(method["return_type"]):
-                    return_statement += "return Ref<" + strip_name(method["return_type"]) + ">(";
+					return_statement += "return Ref<" + strip_name(method["return_type"]) + ">::__internal_constructor(";
```

Comment:
You shouldn't use tabs here 😛

Reply:
Duh, my IDE is configured with tabs because Godot uses tabs... is there a way to configure this per project in QtCreator?